packages/flutter_tools/templates/create/ios-objc.tmpl/Podfile

```diff
@@ -1,36 +0,0 @@
-# Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
-
-if ENV['FLUTTER_FRAMEWORK_DIR'] == nil
-  abort('Please set FLUTTER_FRAMEWORK_DIR to the directory containing Flutter.framework')
-end
-
-target 'Runner' do
-  # Pods for Runner
```

Comment:
Thanks for pointing out. I've been meaning to ask you guys but forgot. 

Can you tell me more about the background for using use_frameworks! on swift projects only? It seems like frameworks are supported for both languages in iOS 8. Is it due to a final app size difference? 

+ @collinjackson for background on https://github.com/flutter/flutter/issues/9694. Would swift users still have the same issue integrating firebase?